ospo-mindmap/Content/ospomindmap.md

```diff
@@ -70,21 +84,80 @@ development team of the Linux Project
 
 - Security Standards
 - License Compliance Policies
-- Tooling
-  - Build in-house
-  - Adopt existing tools
-      - Commercial tools
-      - [Open Source tools](https://oss-compliance-tooling.org/Tooling-Landscape/OSS-Based-License-Compliance-Tools/)
-- Design OSS infrastructure
-
-### OS Community Education
 
-#### Create Internal OS Ambassadors
 
-- Drive event sponsorship
-- Train speakers for coding forums
-  
-#### Create educational programs
+### 📝 Establish and Improve Open Source Policies and Processess
+
+##### Policies
+
+- Using and compliance policy
```

Comment:
Suggestion node name change:
Using and compliance policy --> Using OSS and OSS license compliance policy
Just to make it super clear for everyone.

Reply:
Changed it to ''Using open source and open source compliance policy as for me the compliance includes licensing, security, trademarks, patents, etc.